### ISSUE TITLE
ENOENT when running `lerna bootstrap` on Ubuntu 16

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#1016]
```
{"time":1516902039470,"hostname":"ip-172-31-28-66","pid":2553,"level":"error","name":"pnpm","message":{"errno":-2,"code":"ENOENT","syscall":"open","path":"/home/ubuntu/.pnpm-store/2/registry.npmjs.org/regenerator-runtime/0.11.0_stage/package.json"},"err":{"name":"Error","message":"ENOENT: no such file or directory, open '/home/ubuntu/.pnpm-store/2/registry.npmjs.org/regenerator-runtime/0.11.0_stage/package.json'","code":"ENOENT","stack":"Error: ENOENT: no such file or directory, open '/home/ubuntu/.pnpm-store/2/registry.npmjs.org/regenerator-runtime/0.11.0_stage/package.json'"}}
```

`lerna bootstrap --concurrency 1` didn't have the problem.

Happened whether `pnpm server` is running or not.

### Additional information:

 - `node -v` prints: v8.1.2
 - Windows, OS X, or Linux?: Ubuntu 16

**Comment by zkochan** [comment]
I think this is the only place where this type of error could happen:

https://github.com/pnpm/package-requester/blob/master/src/packageRequester.ts#L440

I don't know how would that be possible though.

**Comment by vjpr** [comment]
Could it also happen in the `renameOverwrite`?

https://github.com/pnpm/package-requester/blob/master/src/packageRequester.ts#L449

**Comment by vjpr** [comment]
`doFetchToStore()` is and async function and is not called with `await`.

**Comment by zkochan** [comment]
renameOverwrite renames the folder not files inside the folder.

The reason I assume that that line of code fails is because it is the only operation that happens in the `_stage` folder before rename happens.

**Comment by vjpr** [comment]
I think I remember seeing a copy command fail once with the same error...

**Comment by zkochan** [comment]
The copying issue that you remember was fixed by @etamponi and released in [1.25.1](https://github.com/pnpm/pnpm/releases/tag/v1.25.1)

**Comment by zkochan** [comment]
This might be fixed in `pnpm@1.31.6`
